File: jupyter2/problems/iteration/gaussian_integral/jl/gaussian_integral.jl
### begin my answer

### end my answer

function main()
    @assert abs(gaussian_integral(1, 1000) - 1.493648) < 1.0e-6
    @assert abs(gaussian_integral(2, 2000) - 1.764163) < 1.0e-6
    @assert abs(gaussian_integral(10, 10000) - sqrt(pi)) < 1.0e-6
    println("OK")
end

main()

File: jupyter2/problems/iteration/gaussian_integral/rs/gaussian_integral.rs
/** begin my answer */

/** end my answer */

fn main() {
    let pi = std::f64::consts::PI;
    assert!((gaussian_integral(1.0, 1000) - 1.493648).abs() < 1.0e-6);
    assert!((gaussian_integral(2.0, 2000) - 1.764163).abs() < 1.0e-6);
    assert!((gaussian_integral(10.0, 10000) - pi.sqrt()).abs() < 1.0e-6);
    println!("OK");
}
